README.md

```diff
@@ -486,3 +486,4 @@ Currently Universe Server provides support for the following versions of DC/OS
 | 1.8                   | Full Support  |
```

Comment:
@jsancio what does Full support mean? Is it ever going to change (Not sure if introduction of package.dcos is gonna change this wording) ?